pkgs/misc/themes/flat-plat/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, gnome_themes_standard }:
+
+stdenv.mkDerivation rec {
+  name = "flat-plat-gtk-theme-${version}";
+  version = "3.20.20160404";
+
+  src = fetchurl {
+    url = "https://github.com/nana-4/Flat-Plat/releases/download/${version}/Flat-Plat-${version}.tar.gz";
+    sha256 = "089qyn1sqilyry4bw4l8v3yi55aw9pakbkp59z6r9y23m4j3mdfb";
+  };
+
+  buildInputs = [ gnome_themes_standard ];
```

Comment:
How `gnome_themes_standard` is used? I mean, build phase is disabled, there is no configure script and install phase doesn't use that either. Is that really needed?